lib/profile_calculator.rb

```diff
@@ -0,0 +1,46 @@
+require "pry"
+
+class ProfileCalculator
+  attr_reader :hash, :points
+
+  ONE_COUNT = 1
+  ZERO_COUNT = 0
+  COMPLETE_PROFILE_COUNT = 5
+
+  def initialize(hash)
+    @points = 0
+    @hash = hash
+    calculate
+  end
+
+  def calculate
+    count = 0
+    count += profile_item_is_complete?("bio", points: 1)
+    count += profile_item_is_complete?("blog", points: 3)
+    count += profile_item_is_complete?("location", points: 1)
+    count += profile_item_is_complete?("email", points: 2)
+    count += profile_item_is_complete?("name", points: 1)
+    profile_is_complete?(count)
+  end
+
+  private
+
+  def profile_is_complete?(count)
```

Comment:
Unused method argument - `count`. If it's necessary, use `_` or `_count` as an argument name to indicate that it won't be used. You can also write as `profile_is_complete?(*)` if you want the method to accept any arguments but don't care about them.